distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -68,57 +79,40 @@ public void beforeExam() throws Exception {
             getServiceManager().createManagedService(cswRegistryStoreProperties.FACTORY_PID,
                     cswRegistryStoreProperties);
             getCatalogBundle().waitForCatalogStore(REGISTRY_CATALOG_STORE_ID);
-
+            destinations = new HashSet<>();
+            destinations.add(REGISTRY_CATALOG_STORE_ID);
         } catch (Exception e) {
             LOGGER.error("Failed in @BeforeExam: ", e);
             fail("Failed in @BeforeExam: " + e.getMessage());
         }
-    }
 
-    @Test
-    public void testCswRegistryIngest() throws Exception {
-        createRegistryEntry("urn:uuid:2014ca7f59ac46f495e32b4a67a51279");
     }
 
     @Test
-    public void testCswRegistryUpdateFailure() throws Exception {
-        String id = createRegistryEntry("urn:uuid:2014ca7f59ac46f495e32b4a67a51280");
-
-        given().body(Library.getCswRegistryUpdate()
-                .replaceAll("urn:uuid:2014ca7f59ac46f495e32b4a67a51276",
-                        "urn:uuid:2014ca7f59ac46f495e32b4a67a51280")
-                .replace("Node Name", "New Node Name")
-                .replace("2016-01-26T17:16:34.996Z", "2014-02-26T17:16:34.996Z")
-                .replaceAll("someUUID", id))
-                .header("Content-Type", "text/xml")
-                .expect()
-                .log()
-                .all()
-                .statusCode(400)
-                .when()
-                .post(CSW_PATH.getUrl());
+    public void testCswRegistryIngest() throws IOException, XPathExpressionException {
```

Comment:
To keep things simple, I would have all the test methods that need an exception in their signature use `throws Exception` instead of listing them all out.